integrationTests/src/test/java/org/sonarqube/gradle/GradleTest.java

```diff
@@ -227,4 +226,24 @@ public void testJaCoCoProperties() throws Exception {
     assertThat(props.getProperty("sonar.jacoco.reportPaths")).contains(baseDir.resolve("build/jacoco/test.exec").toString());
     assertThat(props.getProperty("sonar.coverage.jacoco.xmlReportPaths")).contains(baseDir.resolve("build/reports/jacoco/test/jacocoTestReport.xml").toString());
   }
+
+  @Test
+  public void testProjectWithConfigurationCacheDoubleExecutionsShouldWork() throws Exception {
+    //--configuration cache was brought by release 6.5.0
+    if (getGradleVersion().isGreaterThan("6.5.0")) {
+      runGradlewSonarQubeWithEnv("/java-gradle-configuration-cache", emptyMap(), "-Dsonar.scanner.dumpToFile=asd", "--configuration-cache");
+      RunResult runResult = runGradlewSonarQubeWithEnv("/java-gradle-configuration-cache", emptyMap(), "-Dsonar.scanner.dumpToFile=asd", "--configuration-cache");
```

Comment:
Did we need a dedicated project for this test?

Reply:
Thank you for the catch, there is no need for a dedicated project in that case